src/components/signup/PhoneForm.js

```diff
@@ -130,7 +130,6 @@ class PhoneForm extends React.Component<Props, State> {
                 onBlur={this.onBlur}
                 onSubmitEditing={this.handleSubmit}
                 enablesReturnKeyAutomatically
-                autoFocus
```

Comment:
the same that https://github.com/GoodDollar/GoodDAPP/pull/2379#discussion_r484514916